### DISCUSSION TITLE
In Ad-UI leave interest group information

### DISCUSSION THREAD

**Discussion by iamnewton** [MagniteEngineering/fledge.polyfill#8] [Specification]
How will the browser pass along the interest group information while inside of a rendered ad?  

It’s possible that this could be passed in via the `browser_signals` in the `score_ad` function within the auction.

**Comment by brodrigu** [comment]
FWIW, I didn't expect we would implement this feature in the test.

**Reply by iamnewton** [reply]
sounds good.  If there was a way to tag these discussions that would be next level.
